config.py

```diff
@@ -27,3 +27,6 @@
 SECRET_KEY = "5e3r1t"
 
 SITE_TITLE="Hi:)"
+
```

Comment:
what is the purpose of this extra spaces? read https://www.python.org/dev/peps/pep-0008/ for python coding style